subjects/devops/better-cat/README.md

```diff
@@ -0,0 +1,64 @@
+## better-cat
+
+### Instructions
+
+Create a script `better-cat.sh` that will behave similarly to the cat command, but with additional functionality.
```

Comment:
I would highlight the `cat` - either with the ` or bold/italic - for better readibility

Reply:
Done